File: Cpp/Array/Min and Max in Array.cpp
//https://www.geeksforgeeks.org/problems/find-minimum-and-maximum-element-in-an-array4428/1
pair<long long, long long> getMinMax(long long a[], int n) {
    long long num1=a[0];
    long long num2=a[0];
    for(int i=0;i<n;i++){
        if(num1>a[i]){
            num1=a[i];
        }
        if(num2<a[i]){
            num2=a[i];
        }
    }
    return {num1,num2};
}

File: Python/Array/Min and Max in Array.py
//https://www.geeksforgeeks.org/problems/find-minimum-and-maximum-element-in-an-array4428/1
class Solution:
    def get_min_max(self, arr):
        if not arr:
            return None  # or raise an error, depending on your requirement

        num1 = arr[0]
        num2 = arr[0]
        
        for num in arr:
            if num < num1:
                num1 = num
            if num > num2:
                num2 = num
        
        return (num1, num2)
